sdks/python/apache_beam/runners/interactive/extensions/interactive_beam_side_panel/setup.py

```diff
@@ -0,0 +1,104 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Setup Module to setup Python Handlers for the interactive_beam_side_panel extension.
+"""
+import os
+
+import setuptools
+
+from jupyter_packaging import combine_commands
+from jupyter_packaging import create_cmdclass
+from jupyter_packaging import ensure_python
+from jupyter_packaging import ensure_targets
+from jupyter_packaging import get_version
+from jupyter_packaging import install_npm
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+# The name of the project
+name = "interactive_beam_side_panel"
```

Comment:
package name probably needs to be something like apache_beam_...

Reply:
The name is never set in the stone, so we can decide a naming convention and apply it before we publish them or simply publish under new names.
The package can be published under an org ('scoped package') such as `@apache/beam_interactive_side_panel` or plainly `apache_beam_interactive_side_panel`.